homeassistant/components/sabnzbd/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
```

Comment:
`DATA_SABNZBD`, `DEFAULT_HOST`, `DEFAULT_PORT` and `DEFAULT_SSL` are unused in const.py